src/plugins/ab-testing-service.js

```diff
@@ -33,6 +33,16 @@ function getEnvironment() {
     ? "C"
     : "D"
 
+  // Définition de la valeur d'AB testing pour la refonte de la page de résultats d'une aide
+  ABTesting.benefit_result_page = ABTesting.benefit_result_page || {}
+  ABTesting.benefit_result_page.value = ["C", "D"].includes(
+    ABTesting.benefit_result_page.value
+  )
+    ? ABTesting.benefit_result_page.value
+    : Math.random() > 0.5
+    ? "C"
+    : "D"
```

Comment:
```suggestion
  ABTesting.benefit_result_page = ABTesting.benefit_result_page || {}
  ABTesting.benefit_result_page.value = ABTesting.benefit_result_page.value || (Math.random() > 0.5
    ? "OldUI"
    : "NewUI")
```


La complexité du test AB au dessus est spécifique à la transformation d'un premier test en deux nouvelles variantes. Ça peut être simplifié ici.